### PR TITLE
Rasos performance gain

### PR DESCRIPTION
**Pull request by rasos**
performance tweaks tested on Ubuntu 18.04

### CODE REVIEW

**Comment by rasos** [comment]
fixes #260

**Comment by ffdixon** [comment]
This is really interesting -- can you be more specific on the improvements you encountered?

**Comment by rasos** [comment]
We run a few BBB instances, two of them under quite heavy load.      
We've applied a low latency kernel as we always do with video / audio related projects, including some Jitsi videobridges.        
For BBB, we never had an issue with freeswitch suddenly disappearing nor audio crackling sound.    
We did not perform any load / selenium testing though to check how many hundred users / streams can be served. Bottleneck with NodeJS will remain, though.

**Comment by ffdixon** [comment]
To confirm, you you applied these changes to 18.04 using BigBlueButton 2.3-dev -- right?  Was this on a bare-metal server or virtual server?

**Comment by rasos** [comment]
We have applied a low latency kernel both on BBB 2.2 on Ubuntu 16.04 and on BBB 2.3-dev on Ubuntu 18.04 (the latter not in production use yet). All our BBB servers run on bare metal.

**Comment by basisbit** [comment]
@rasos did you experience the problems which this change is supposed to fix/improve on the same environment and same version of bbb before applying these changes? On paper, the low-latency kerne should make next to no difference for a typical BBB setup.

I'd suggest to do some reproducible testing and collect some data.

**Comment by kepstin** [comment]
If we do end up making these changes, we should prefer to use the `linux-lowlatency-hwe-16.04` kernel package in order to get the newer 4.15 kernel backported from ubuntu 18.04 instead of the 4.4 kernel from ubuntu 16.04.

File: bbb-install.sh [hunk 1]
```diff
@@ -306,6 +306,14 @@ HERE
 
   apt-get update
   apt-get dist-upgrade -yq
+  
+  # improve performance with low latency kernel, higher limits and larger networking buffer | issue #260
+  # roland.alton@fairkom.eu 2020-08-20 BBB hackathon
+  apt-get install linux-lowlatency linux-headers-lowlatency
+  echo -e "DefaultLimitNOFILE=65000\r\nDefaultLimitNPROC=65000\r\nDefaultTasksMax=65000" | sudo tee -a /etc/systemd/system.conf
```
Comment (basisbit) [inline bbb-install.sh:313]: this line should be removed. Proper limits are already set in the services configurations. The way how it is done here would not work for current linux systems.

**Comment by ffdixon** [comment]
This pull request is old and, I believe, most users are deploying BigBlueButton using the default kernels.

**Pull request closed without merging**